client/src/App.js

```diff
@@ -36,45 +36,45 @@ function App() {
         <footer className="fixed bg-gray-800 w-full bottom-0 items-center flex flex-full flex-col py-20">
           <address class="flex space-x-10">
             <div>
-              <a href="#">
+              {/* <a href="#"> */}
```

Comment:
You can actually leave these # hrefs in here. It's not valid to push to production, but for placeholders it's ok